CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.3.0
+
+* Introduce a new interface `IriPlainIdentifierAwareConverterInterface`
```

Comment:
This hasn't been updated to reflect the changes. 😜

Reply:
and btw we never change the changelog in merge requests it's done when releasing only

Reply:
Not sure if that's a good idea, to be honest. For example, Symfony requires updating the changelog: https://raw.githubusercontent.com/symfony/symfony/master/.github/PULL_REQUEST_TEMPLATE.md

Reply:
That's why I updated it, I've contributing some PRs to symfony lately and add to update it ;).

Reply:
And I think it's a good practice too to update the changelog when adding new things or deprecating things so we gain time when releasing